airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -235,14 +236,15 @@ private static void migrateExistingConnection(final ConfigRepository configRepos
       throws JsonValidationException, ConfigNotFoundException, IOException {
     LOGGER.info("Start migration to the new scheduler...");
     final Set<UUID> connectionIds =
-        configRepository.listStandardSyncs().stream().map(standardSync -> standardSync.getConnectionId()).collect(Collectors.toSet());
+        configRepository.listStandardSyncs().stream()
+            .filter(standardSync -> standardSync.getStatus() == Status.ACTIVE || standardSync.getStatus() == Status.INACTIVE)
```

Comment:
What is the reasoning behind filtering down to only ACTIVE and INACTIVE?

Reply:
ACTIVE = enabled workflow

Reply:
I think that makes sense to me. Though we should probably get a confirmation from @jrhizor to be sure